wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -65,6 +67,12 @@ def _check_exception_type(self, node: ast.Raise) -> None:
                 BaseExceptionRaiseViolation(node, text=exception_name),
             )
 
+    def _check_bare_raise(self, node: ast.Raise) -> None:
+        parent_except = walk.get_closest_parent(node, ast.ExceptHandler)
```

Comment:
We should first check that `node.exc is None`.
Because, this operation is quite heavy, we don't waste it on `raise` with arguments

Reply:
Ah yeah, that makes a lot of sense. Thanks!